lighthouse-core/gather/gatherers/styles.js

```diff
@@ -115,9 +115,7 @@ class Styles extends Gatherer {
       Promise.all(contentPromises).then(styleHeaders => {
         driver.off('CSS.styleSheetAdded', this._onStyleSheetAdded);
         driver.off('CSS.styleSheetRemoved', this._onStyleSheetRemoved);
-        return driver.sendCommand('CSS.disable')
```

Comment:
Not sure if this is the right time to pull out domain information when multiple gatherers use the same domain, but for now just manually removing here.